pandas/tests/groupby/test_apply.py

```diff
@@ -1177,3 +1177,34 @@ def test_apply_empty_string_nan_coerce_bug():
         index=MultiIndex.from_tuples([(1, ""), (2, "")], names=["a", "b"]),
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("is_float", [True, False])
```

Comment:
You can parametrize over the values Not a flag, e.g. values, [[1,2,3], [1.0,2.0,3.0]]

Reply:
Oh sure, how about now?